packages/aws-cdk-lib/cx-api/lib/features.ts

```diff
@@ -1034,6 +1036,41 @@ export const FLAGS: Record<string, FlagInfo> = {
     introducedIn: { v2: 'V2NEXT' },
     recommendedValue: true,
   },
+
+  //////////////////////////////////////////////////////////////////////
+  [ENABLE_ALBV2_EXTERNALAPPLICATIONLISTENER_ADDTARGETGROUP_TO_ADDACTION_MIGRATION]: {
```

Comment:
I don't think we need to add `ENABLE` to the name here, also I feel like `MIGRATE` suggests future changes that might not necessarily be coming (if that makes sense). 

```suggestion
  [ALBV2_EXTERNALAPPLICATIONLISTENER_SWITCH_FROM_ADDTARGETGROUP_TO_ADDACTION]: {
```